be/src/exec/pipeline/aggregate/spillable_aggregate_blocking_sink_operator.cpp

```diff
@@ -39,10 +39,10 @@ bool SpillableAggregateBlockingSinkOperator::is_finished() const {
 }
 
 Status SpillableAggregateBlockingSinkOperator::set_finishing(RuntimeState* state) {
+    auto defer_set_finishing = DeferOp([this]() { _aggregator->spill_channel()->set_finishing(); });
```

Comment:
what if NO_SPILL? this line should after L:48?

Reply:
spill_channel should always be called